src/app/panels/terminal.js

```diff
@@ -405,24 +408,22 @@ class Terminal extends BaseApi {
     self._cmdIndex = -1
     self._cmdTemp = ''
 
-    var intro = yo`
-      <div><div> - Welcome to Remix ${packageV.version} - </div><br>
-      <div>You can use this terminal for: </div>
-      <ul class=${css2.ul}>
-        <li>Checking transactions details and start debugging.</li>
-        <li>Running JavaScript scripts. The following libraries are accessible:
-          <ul class=${css2.ul}>
-            <li><a target="_blank" href="https://web3js.readthedocs.io/en/1.0/">web3 version 1.0.0</a></li>
-            <li><a target="_blank" href="https://docs.ethers.io/ethers.js/html/">ethers.js</a> </li>
-            <li><a target="_blank" href="https://www.npmjs.com/package/swarmgw">swarmgw</a> </li>
-            <li>remix (run remix.help() for more info)</li>
-          </ul>
-        </li>
-        <li>Executing common command to interact with the Remix interface (see list of commands above). Note that these commands can also be included and run from a JavaScript script.</li>
-        <li>Use exports/.register(key, obj)/.remove(key)/.clear() to register and reuse object across script executions.</li>
-      </ul>
-      </div>
-    `
+    var intro = yo`<div><div> - Welcome to Remix ${packageV.version} - </div><br>
```

Comment:
why?

Reply:
that's reformatting issue, will rollback that